boards/arm/bl654_dvk/doc/bl654_dvk.rst

```diff
@@ -0,0 +1,180 @@
+.. _bl654_dvk:
+
+BL654_DVK
+#################
+
+Overview
+********
+
+Zephyr applications use the bl654_dvk board configuration
```

Comment:
We're defining some new intro paragraph guidelines that will let us generate a better board index page, so let's change this to:
```
The BL654 Development Kit hardware provides
support for the Laird BL654 module powered by a Nordic Semiconductor nRF52840 ARM Cortex-M4F CPU.

This development kit has the following features:
```